provider/aws/aws.go

```diff
@@ -684,6 +684,10 @@ func (p *AWSProvider) AdjustEndpoints(endpoints []*endpoint.Endpoint) []*endpoin
 		}
 
 		if alias {
+			if ep.RecordTTL.IsConfigured() {
+				log.Debugf("Modifying endpoint: %v, setting ttl=%v", ep, recordTTL)
+				ep.RecordTTL = recordTTL
```

Comment:
Would it be better to unconfigure the TTL instead?

Reply:
It does sound more robust, updated!

Reply:
I actually had to revert that as the tests in `TestAWSRecords` were failing, it also seem more consistent with what `records()` does:
https://github.com/kubernetes-sigs/external-dns/blob/0019d158dc448f8987285ef49b90432af5e91cee/provider/aws/aws.go#L398

Reply:
Ah, right